test/README.md

```diff
@@ -147,6 +148,9 @@ environment.
    lima version 1.0.0 or later is required, latest version is
    recommended.
 
+   > [!NOTE]
+   > Tested with lima version 1.0.4.
```

Comment:
This does not work when indented under a list. Just add the text at the end of the paragraph on line 147.